src/backend/distributed/planner/multi_explain.c

```diff
@@ -182,6 +182,8 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 		return;
 	}
 
+	ExplainOpenGroup("Distributed Query", NULL, true, es);
```

Comment:
So, with that I'd expect `MasterQuery` to have `true` since there could be a single `MasterQuery`? But, I see that it is false. Is there anything special about it?